db.go

```diff
@@ -397,6 +400,17 @@ func Open(opt Options) (*DB, error) {
 	return db, nil
 }
 
+// logToTestChan sends a message to the DB's testChan. Note that we expect
+// that the DB *never* closes this channel; the responsibility for
+// allocating and closing the channel belongs to the test module.
+// if db.testChan is nil or has never been initialized, this will
+// be silently ignored.
+func (db *DB) logToTestChan(msg string) {
```

Comment:
This function should have names based on what it does, something notifyUpdateDiscardStats or something

Reply:
Well, what it literally does is log a message to the test channel.  :)
If we ever wish to extend the usage of testChan (or "syncChan" if you prefer) to cover other use-cases, we should avoid having it be related specifically to updateDiscardStats. Having said that, it is worth noting that this new func is referenced in 2 places in the code: first to update for discardStats and second to notify when we reach the end of initialization of the valueLog (db.vlog). It is *not* actually used to grab the updateDiscardStats! The discardStats are retrieved when we close the database (or more precisely, the valueLog).